contracts/metagov/VlAuraFishyImplementation.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import "./IAuraLocker.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Votes.sol";
+
+// Hidden Hand Aura bribe contract
+interface IAuraBribe {
+    function setRewardForwarding(address to) external;
+
+    function getBribe(
+        bytes32 proposal,
+        uint256 proposalDeadline,
+        address token
+    ) external view returns (address bribeToken, uint256 bribeAmount);
+}
+
+/// @author eswak
+contract VlAuraFishyImplementation {
+    address public constant AURA = 0xC0c293ce456fF0ED870ADd98a0828Dd4d2903DBF;
+    address public constant VLAURA = 0x3Fa73f1E5d8A792C80F426fc8F84FBF7Ce9bBCAC;
+
+    // TODO: Update this before deploying, Fishy.
```

Comment:
Flagging this TODO

Reply:
it was for him to try the tests on his own ^^